<launch>
<!-- Launch file for Livox AVIA LiDAR -->

	<arg name="rviz" default="true" />

	<arg name="param_file" default="$(find fast_calib)/config/qr_params.yaml"/>
	<rosparam file="$(arg param_file)" command="load" subst_value="true"/>

	<node pkg="fast_calib" type="fast_calib" name="fast_calib" output="screen">
	</node>
	
	<group if="$(arg rviz)">
		<node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_calib)/rviz_cfg/fast_livo2.rviz" />
	</group>
launch-prefix="gdb -ex run --args"
launch-prefix="valgrind --leak-check=full"
</launch> 	
